cucumber/authenticators_jwt/features/authn_jwt_fetch_identity_from_url.feature

```diff
@@ -0,0 +1,178 @@
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+
+    - !host
+      id: some_policy/host_test_from_url
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host some_policy/host_test_from_url
+
+    - !user
+      id: user_test_from_url@some_policy
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !user user_test_from_url@some_policy
+
+    - !user
+      id: user_test_from_url
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !user user_test_from_url
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+
+  Scenario: User send in URL, user not in root, 200 ok
+    Given I have a "variable" resource called "test-variable"
+    And I permit user "user_test_from_url@some_policy" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with user_test_from_url%40some_policy account in url
+    Then user "user_test_from_url@some_policy" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:user:user_test_from_url@some_policy successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: Host send in URL, host not in root, 200 ok
+    Given I have a "variable" resource called "test-variable"
+    And I permit host "some_policy/host_test_from_url" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with host%2Fsome_policy%2Fhost_test_from_url account in url
+    Then host "some_policy/host_test_from_url" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:some_policy/host_test_from_url successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
```

Comment:
add log code

Reply:
This is success message not sure it has log code 

![image](https://user-images.githubusercontent.com/73157235/122670166-3b646900-d1c9-11eb-8261-9ce97d447c8a.png)